<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationDialog</class>
 <widget class="QDialog" name="ConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>設定画面</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>340</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="pictureQualityLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>描画クォリティ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="motionQualityLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>動画クォリティ：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="pictureHighRadioButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>高</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">pictureButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="pictureLowRadioButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>低</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">pictureButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="motionHighRadioButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>高</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">motionButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="motionMiddleRadioButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>中</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">motionButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="motionLowRadioButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>低</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">motionButtonGroup</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="pictureButtonGroup"/>
  <buttongroup name="motionButtonGroup"/>
 </buttongroups>
</ui>
